portfolio/src/main/webapp/contact.html

```diff
@@ -39,8 +39,11 @@ <h3>Contact me:</h3>
     <textarea name="messageInput" placeholder="Leave a message here" rows="6" cols="50"></textarea>
     <br/>
     <input type="submit" value="Submit"/>
-
+    <br><br>
+    <p>See previous messages <a href="/messages">here</a>.</p>
```

Comment:
I think you need to redeploy your webapp. This element doesn't show up on https://ywu-sps-summer22.appspot.com/contact.html.